meta-mender-raspberrypi/recipes-bsp/u-boot/u-boot-raspberrypi.inc

```diff
@@ -6,9 +6,12 @@ BOOTENV_SIZE:rpi ?= "0x4000"
 # use of boot.scr and cmdline.txt.
 MENDER_UBOOT_AUTO_CONFIGURE:rpi = "0"
 
+SRC_URI:append:rpi = " file://memory_size.cfg"
```

Comment:
Isn't this configuration uboot specific?
```suggestion
SRC_URI:append:rpi:mender-uboot = " file://memory_size.cfg"
```

Reply:
@lluiscampos it is `u-boot` specific, hence in the `u-boot` recipe, but its not Mender specific. So not tied to the `mender-uboot` feature.

Technically both of those *could* go to `meta-raspberrypi`, but I want to unblock our users first, then care about upstream.